### PR TITLE
Reports summary

### LINKED ISSUE #1329: Reports
Reports view includes three different sections: **Summary**, **Bar chart** and **Pie chart** with a list. In this issue implement the view **only with the Summary section**.

By default, we show data from the default workspace and of the single user only. *But keep in mind we want to add WS selection in reports page as well if a user has more than one workspace.*

- [x] Total and billable sections should always have 16dp padding between each other. This means sections itself are responsive and their width depends on the screen size.
- [x] Billable bar/chart is dynamic.
- [ ] Total hours chart is static.

### Reports
reports

Zeplin: https://zpl.io/2yBEpQL

### PR DESCRIPTION
**Pull request by willsb**
Closes #1329 

The illustration for the total chart is missing from Zeplin. I need it in all android supported dimensions. Ping @pe0ny.

If those are not supplied by Monday, I think we can still merge this and open an issue to add the illustrations in a different PR.

### CODE REVIEW

File: Toggl.Giskard/Adapters/ReportsRecyclerAdapter.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        public ReportsRecyclerAdapter(IntPtr javaReference, JniHandleOwnership transfer)
+            : base(javaReference, transfer)
+        {
+        }
+
+        public override int ItemCount => 1;
+
+        public override object GetItem(int viewPosition) => ViewModel;
+    }
+}
```
Comment (simonrozsival) [inline Toggl.Giskard/Adapters/ReportsRecyclerAdapter.cs]: \n

File: Toggl.Giskard/Resources/values/Colors.xml [hunk 2]
```diff
@@ -23,4 +23,5 @@
     <color name="switchTint">#328FFF</color>
     <color name="launcherBackgroundColor">#FFFF</color>
     <color name="checkboxTint">#328FFF</color>
+	<color name="reportsLabelColor">#757575</color>
```
Comment (simonrozsival) [inline Toggl.Giskard/Resources/values/Colors.xml]: Tabs?

File: Toggl.Giskard/Views/ReportsRecyclerView.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            : this(context, attrs, 0)
+        {
+        }
+
+        public ReportsRecyclerView(Context context, IAttributeSet attrs, int defStyle)
+            : base(context, attrs, defStyle, new ReportsRecyclerAdapter())
+        {
+        }
+    }
+}
```
Comment (simonrozsival) [inline Toggl.Giskard/Views/ReportsRecyclerView.cs]: \n

File: Toggl.Giskard/Resources/layout/ReportsActivitySummary.axml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+                android:layout_alignStart="@+id/ReportsSummaryBillable">
+                <View
+                    local:MvxBind="WidthPercentage BillablePercentage"
+                    android:background="#328FFF"
+                    android:layout_height="match_parent"
+                    android:layout_width="match_parent" />
+            </FrameLayout>
+        </RelativeLayout>
+    </android.support.v7.widget.CardView>
+</FrameLayout>
```
Comment (simonrozsival) [inline Toggl.Giskard/Resources/layout/ReportsActivitySummary.axml]: Is this a generated file? If not, there should be a \n here.

File: Toggl.Giskard/Activities/ReportsActivity.cs [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+{
+    [MvxActivityPresentation]
+    [Activity(Theme = "@style/AppTheme",
+              ScreenOrientation = ScreenOrientation.Portrait,
+              ConfigurationChanges = ConfigChanges.Orientation | ConfigChanges.ScreenSize)]
+    public sealed class ReportsActivity : MvxAppCompatActivity<ReportsViewModel>
+    {
+        protected override void OnCreate(Bundle bundle)
+        {
+            this.ChangeStatusBarColor(Color.ParseColor("#2C2C2C"));
```
Comment (simonrozsival) [inline Toggl.Giskard/Activities/ReportsActivity.cs]: Why isn't this color value in some `Constants` class?

**Comment by pe0ny** [comment]
@willsb I updated zeplin and total chart illustration is now available :)

File: Toggl.Giskard/Activities/ReportsActivity.cs [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+using Toggl.Foundation.MvvmCross.ViewModels;
+using Toggl.Giskard.Extensions;
+
+namespace Toggl.Giskard.Activities
+{
+    [MvxActivityPresentation]
+    [Activity(Theme = "@style/AppTheme",
+              ScreenOrientation = ScreenOrientation.Portrait,
+              ConfigurationChanges = ConfigChanges.Orientation | ConfigChanges.ScreenSize)]
+    public sealed class ReportsActivity : MvxAppCompatActivity<ReportsViewModel>
```
Comment (kornelijepetak) [inline Toggl.Giskard/Activities/ReportsActivity.cs]: This class should provide a way to return to the main screen via `back` button?

File: Toggl.Giskard/Resources/layout/ReportsActivity.axml [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+        <Button
+            local:MvxBind="Click OpenSettingsCommand"
+            android:background="@drawable/settings_dark"
+            android:layout_gravity="end"
+            android:layout_marginLeft="16dp"
+            android:layout_marginRight="16dp"
+            android:layout_width="24dp"
+            android:layout_height="24dp" />
+        <ImageView
+            android:src="@drawable/timer_light"
```
Comment (kornelijepetak) [inline Toggl.Giskard/Resources/layout/ReportsActivity.axml]: The `timer` button does not return to the timer screen.

File: Toggl.Giskard/Resources/layout/ReportsActivitySummary.axml (hunk 4 above)
Comment (willsb) [reply Toggl.Giskard/Resources/layout/ReportsActivitySummary.axml]: It gets stripped by the editor, so we agreed to not add those

File: Toggl.Giskard/Activities/ReportsActivity.cs (hunk 5 above)
Comment (willsb) [reply Toggl.Giskard/Activities/ReportsActivity.cs]: I agree it should. We need to refactor all of those at once, tho

Comment (simonrozsival) [reply Toggl.Giskard/Activities/ReportsActivity.cs]: OK

**Pull request merged**